app/modules/users/models.py

```diff
@@ -233,19 +235,6 @@ def __repr__(self):
             ')>'.format(class_name=self.__class__.__name__, self=self)
         )
 
-    @classmethod
-    def suggest_password(cls):
-        from xkcdpass import xkcd_password as xp
```

Comment:
I agree that this function can now be killed.  Since we are removing all of the references to `xkcdpass`, lets go ahead and remove it from the installed packages list.  In Python, this is generally managed by a `requirements.txt` file.  This runtime dependency is listed in `app/requirements.txt:L59` and can be dropped.